.github/scripts/render-portfolio.ps1

```
@@ -28,6 +28,45 @@ param(
+    param([string[]]$labels)
+    if ($labels -contains 'priority: high')   { return 0 }
+    if ($labels -contains 'priority: medium') { return 1 }
+    if ($labels -contains 'priority: low')    { return 2 }
+    return 3  # unlabeled / last
+}
+
+# Returns -Ticks for descending createdAt sort, or 0 when createdAt is absent/unparseable.
+function Get-CreatedAtSortTicks {
+    param([string]$createdAt)
+    if ($createdAt) {
+        try { return -[datetime]::Parse($createdAt).Ticks } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)